internal/kafka/internal/config/kafka_supported_instance_types.go

```diff
@@ -174,3 +174,19 @@ func (p Plan) GetSizeID() (string, error) {
 	}
 	return t[1], nil
 }
+
+type Quantity string
+
+func (q *Quantity) String() string {
+	return string(*q)
+}
+
+func (q *Quantity) ToFloat32() (float32, error) {
+	var output float32
+	p, err := resource.ParseQuantity(q.String())
```

Comment:
Invoking `q.String()` here generates a copy of the string. The string is then copied again when passed to `ParseQuantity`.
We can avoid one copy by simply casting `*q` to `string` here

Reply:
Thanks @ziccardi I'll make this change